R/add_data.R

```diff
@@ -555,9 +557,12 @@ addDataDensity <- function(dtOld, dataDist, varname, uselimits = FALSE) {
   dataDist <- round(dataDist, 0)
   
   if (uselimits) {
-    density_est <- stats::density(dataDist, n = 10000, from = min(dataDist), to = max(dataDist))
+    density_est <- 
+      stats::density(dataDist, n = 10000, from = min(dataDist, na.rm = TRUE), 
+                     to = max(dataDist, na.rm = TRUE), na.rm = TRUE)
```

Comment:
This will set `na.rm` always to true here, given that it was always `FALSE` before I assume you'd want it to be based on the user input?

Reply:
Yes - that is by design. I generate the data without missing data based on the non-missing data specification, and then if na.rm is set to FALSE, I randomly allocate missing values.